form.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang=en>
+ <head>
+  <title>Test</title>
+ </head>
+ <body>
+
+<form action="/action_page.php">
+
+    <fieldset>
+      <legend>Basic User Information:</legend>
+      <label for="fname">Full Name:</label>
+      <input type="text" id="fname" name="fname" required><br>
+
+      <label for="eml">Email:</label>
+      <input type="email" id="eml" name="eml" required><br>
+
+      <label for="pswd1">Password:</label>
+      <input type="password" id="pswd1" name="pswd1" required>      
+      <label for="pswd2">Confirm Password:</label>
+      <input type="password" id="pswd2" name="pswd2" required><br>
+      
+ 
+      <span>Gender:</span>
+      <input type="radio" id="gender1" name="gender1" value="Male">
+      <label for="gender1">Male</label>
+      <input type="radio" id="gender2" name="gender2" value="Female">
+      <label for="gender2">Female</label>
+      <input type="radio" id="gender3" name="gender3" value="Other">
+      <label for="gender3">Other</label><br>
+
+      <label for="DOB">Date of Birthday:</label>
```

Comment:
Avoid using uppercase in attribute values. Better use "dob" instead of "DOB". Even better "date-of-birth" 
Try to use explicit names in order not to get lost in all the abbreviations when the project gets bigger

Reply:
Thank you for the review and your advice!
I also found the information about that password validation requires JavaScript.

I've corrected attribute value to "date-of-birth" and also added the number for the range slider. Honestly, I didn't know how to add this number (although you gave the hint), so I used ChatGPT :) 

I've checked once again with W3C Markup Validation and everything is fine

![Снимок экрана 2023-08-17 130137](https://github.com/HappyJuliana/Tekwill_Nice_1/assets/128743829/faf61f49-4a8c-49d2-8f63-431d3687284b)

Reply:
ChatGPT is a nice guy!